.github/workflows/maven-regression.yaml

```diff
@@ -5,12 +5,13 @@ on:
     branches: [ master ]
   workflow_dispatch:
 
+
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [ "1.4", "1.5", "1.6", "1.7", "1.8", "1.9", "1.10", "1.11", "1.12" ]
+        version: [ "1.5", "1.6", "1.7", "1.8", "1.9", "1.10", "1.11", "1.12" ]
```

Comment:
Why `1.4` is gone?

Reply:
It duplicates integration-tests.yaml workflow, same with latest 1.13 (default, if no profile specified)